x-pack/test/functional/services/browser_exec_scripts/read_pipeline_list_rows.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function readPipelineListRows(container, cssSelectors) {
```

Comment:
Note, I moved this fn to a file in a `browser_exec_scripts` directory, which signals to the linter that this file has special syntax restrictions, since it is executed in the browser with only the transpilation necessary to run the code in node.js.